app/controllers/admin/versions_controller.rb

```diff
@@ -1,17 +1,17 @@
 module Admin
   class VersionsController < Admin::BaseController
-    skip_authorization_check
+    load_resource :conference, find_by: :short_title
+    load_and_authorize_resource class: PaperTrail::Version
 
     def index
-      authorize! :index, PaperTrail::Version.new(item_type: 'User')
-      conf_ids_for_organizer = current_user.is_admin? ? Conference.pluck(:id) : Conference.with_role(:organizer, current_user).pluck(:id)
-      @versions = PaperTrail::Version.where(["conference_id IN (?) OR item_type = 'User'", conf_ids_for_organizer])
+      @conf_ids_with_role = current_user.is_admin? ? Conference.pluck(:short_title) : Conference.with_role([:organizer, :cfp, :info_desk], current_user).pluck(:short_title)
+
+      return if @conference.blank?
```

Comment:
Can `@conference` really be blank? If it's not found by `load_resource`, then you get a 404.